packages/manager/modules/logs-to-customer/src/data/hooks/useLogTailMessages.ts

```diff
@@ -0,0 +1,78 @@
+import { useContext, useEffect, useRef, useState } from 'react';
+import { useQuery, useQueryClient } from '@tanstack/react-query';
+import { TemporaryLogsLink } from '../types/dbaas/logs';
+import { getLogTailMessages, Tmessage } from '../api/logTailMessages';
+import { LogsContext } from '../../LogsToCustomer.context';
+import { logMessagesMock } from '../mocks/logMessage.mock';
+
+interface IUseLogTailMessages {
+  logTailMessageUrl: TemporaryLogsLink['url'];
+}
+
+export const getLogTailMessageQueryKey = (logTailMessageUrl: string) => [
+  'getLogTailMessages',
+  logTailMessageUrl,
+];
+
+export const useLogTailMessages = ({
+  logTailMessageUrl,
```

Comment:
Using an object may be overkill passing it as param would be sufficient i think ? 
export const useLogTailMessages = (logTailMessageUrl: TemporaryLogsLink['url']) => { }